docker/Dockerfile

```diff
@@ -1,44 +1,39 @@
-FROM python:3.7.2-alpine3.8
-
-ENV PYTHONUNBUFFERED=1
-ENV PYTHONDONTWRITEBYTECODE=1
+FROM python:3.7.3-slim as base
 ENV PATH="/root/.poetry/bin:$PATH"
 ENV FLASK_APP=search
-
 WORKDIR /srv/app
-
-RUN rm -rf /var/cache/apk/* && \
-    rm -rf /tmp/* && \
-    apk update && \
-    apk add --no-cache \
-    bash \
-    grep \
-    libxslt-dev
-
-RUN apk add --no-cache --virtual .build-dependencies \
-    curl \
-    && \
-    curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python && \
-    poetry config settings.virtualenvs.create false && \
-    apk del .build-dependencies
-
 COPY ./pyproject.toml ./poetry.lock ./
-
-ARG env
-ENV FLASK_ENV=${env}
-
-ARG dev
-ARG simulate_removal
-RUN apk add --no-cache --virtual .build-dependencies \
-    gcc \
-    git \
-    linux-headers \
-    musl-dev \
-    && \
-    poetry install ${dev} && \
-    apk del ${simulate_removal} .build-dependencies
-
 COPY search/ search/
-
+RUN set -ex \
+    && apt-get update -yq \
+    && apt-get install -yq --no-install-recommends curl \
+    && curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python \
+    && poetry config settings.virtualenvs.create false \
+    && apt-get remove --purge --autoremove -yq curl
```

Comment:
to be fair leaving `curl` around may help with healthchecks. But the published image exposes the WSGI protocol rather than HTTP, so not really useful.

Reply:
Should I leave `curl` installed?